services/frontend-service/pkg/cmd/server.go

```diff
@@ -228,12 +226,7 @@ func RunServer() {
 						return
 					}
 				}
-				if strings.HasPrefix(req.URL.Path, "/v2/home") {
-					http.ServeFile(resp, req, "build/index.html")
-				} else {
-					mux.ServeHTTP(resp, req)
-				}
-
+				mux.ServeHTTP(resp, req)
```

Comment:
it doesn't work like this, the http server in go is configured to convert URLs to paths in order to return responses, there is nothing in the path `v2/home` on disk so it will return 404, the current config routes `/v2/home` to `build/index.html` which is generated by react and it handles routing